tests/test_pyi_files.py

```diff
@@ -42,23 +42,42 @@ def test_pyi_file(path: str) -> None:
         option = flag.split("=")[0]
         assert option != "--ignore", bad_flag_msg
 
+    # Silence DeprecationWarnings from our dependencies (pyflakes, flake8-bugbear, etc.)
+    #
+    # For DeprecationWarnings coming from flake8-pyi itself,
+    # print the first occurence of each warning to stderr.
+    # This will fail CI the same as `-Werror:::pyi`,
+    # but the test failure report that pyflakes gives is much easier to read
+    # if we use `-Wdefault:::pyi`
+    flake8_invocation = [
+        sys.executable,
+        "-Wignore",
+        "-Wdefault:::pyi",
+        "-m",
+        "flake8",
+        "-j0",
+    ]
```

Comment:
(I've manually checked that this causes CI to fail if we have DeprecationWarnings that are emitted from flake8-pyi itself)